src/PerfView/PerfView.csproj

```diff
@@ -89,6 +92,7 @@
     <PackageReference Include="Microsoft.IdentityModel.Abstractions" Version="$(MicrosoftIdentityModelAbstractionsVersion)" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="$(MicrosoftIdentityModelTokensVersion)" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="$(MicrosoftIdentityModelJsonWebTokensVersion)" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.Web.WebView2" Version="$(MicrosoftWebWebView2Version)" GeneratePathProperty="true" />
```

Comment:
The reason that we set `GeneratePathProperty=true` for most of these package references is because we copy files from them into the self-extracting PerfView.exe.  I see that we aren't doing that with WebView2.  Are we guaranteed to have WebView2 on the machine already?

Reply:
This should now be all fully resolved, it now includes the WebView DLLs + the drivers as embedded resources. I've verified it is working with the self-extracting PerfView.exe on my machine and it seems to be all working fine.